src/mongoose_factory.coffee

```diff
@@ -39,7 +39,14 @@ buildDefinitionFromSchemaType = (schemaType, mongoose, {ignoreRequired} = {}) ->
       -> faker.lorem.words()
 
     when schemaType instanceof mongoose.SchemaTypes.Number
-      -> faker.random.number 100
+      ->
```

Comment:
Awesome. I think eventually ideally we would have some sort of shared helper like `unionized.range`... but all in good time.